policies/gitlab/repository.rego

```diff
@@ -246,7 +246,8 @@ repository_allows_committer_approvals_policy {
 # custom:
```

Comment:
not your code, but I think the title and description here should be taken from the GH version, it's more detailed

Reply:
Done